src/components/layout/Header.tsx

```
@@ -15,9 +16,19 @@ type NavItem = {
+  const [isScrolled, setIsScrolled] = useState<boolean>(false);
+  const [isMobileMenuOpen, setIsMobileMenuOpen] = useState<boolean>(false);
+  const [activeSection, setActiveSection] = useState<string>('');
+  const [isNotHome, setIsNotHome] = useState<boolean>(false);
+
+  const params = useParams();
+  const router = useRouter();
+
+  useEffect(() => {
+    if (params && params.id) {
+      setIsNotHome(true);
+    }
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)